### ISSUE TITLE
Cannot roll over

### ISSUE THREAD

**Issue by randombaldguy** [explorer-01-2018/beinghuman#1]
### Steps to reproduce
Rotate child instance of class human by 180 degrees along z-axis. Will not automatically assume original position. 
_(ignore timestamp, this is from 26 feb)_

**Comment by explorer-01-2018** [comment]
yes, this is an issue. but not the most pressing. expect fix around milestone 0.4 or 0.5
_(ignore timestamp, this reply is from 26 of feb.)_
